### ISSUE TITLE
Allow item's description ending with back-tick

### ISSUE THREAD

**Issue by kdeldycke** [sindresorhus/awesome-lint#96]
I'm maintaining an awesome list on managing engineering teams at: https://github.com/kdeldycke/awesome-management

I'd like one day to graduate this draft and propose it to the [canonical awesome repository](https://github.com/sindresorhus/awesome). To prepare that I need to have the linter validates its content. If the majority of reported errors are fixable, I still stumble upon one issue.

To describe an item, I often quote the original content. Here is an example, in which the following Markdown:

```markdown
- [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - `If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.`
```

is rendered to:

- [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - `If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.`

`awesome-lint` reports this as a blocking issue:

```
✖  XX:XX   List item description must end with proper punctuation   remark-lint:awesome-list-item
```

For the linter to pass, I need to add a final point, after the back-tick:

```markdown
- [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - `If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.`.
```

Which renders to:

- [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - `If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.`.

I don't know about typographic dogma, but that description ending with 2 points (one quoted, one not) looks unpleasant.

What should be the rule here?
1. Keep it that way? (i.e. repeating the final point)
2. Fine-tune `awesome-lint` to allow for an item's description to end with a back-tick?

**Comment by sindresorhus** [comment]
I don't think backtick is a good way to do inline quotes. It looks quite noisy. I would rather do:

> - [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - “If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.”

So we could support `”`.

**Comment by kdeldycke** [comment]
Thanks @sindresorhus for the proposal. Literal “...” quotes are indeed appropriate and I agree with you, these makes inline quotes cleaner and less noisy. I'll adopt them for now on! :)

Now this doesn't really address the core issue which is how to properly terminate descriptions made of quotes. But according the [The Punctuation Guide](https://www.thepunctuationguide.com/quotation-marks.html#adjacent-punctuation), ending the sentence without a comma is the correct form.

So I guess it's OK for the linter to allow (or better, enforce) the following notation you're proposing above.

I just double-checked today and yes, that form:

```markdown
- [Teams are like bread](https://blog.jessitron.com/2019/06/15/teams-are-like-bread/) - “If you have one team where the magic is flourishing, don't kill it. Feed it, grow it, and let it be a source of further strong teams. No rushing.”
```

still produces the following error:

```
  ✖   XX:XX   List item description must end with proper punctuation                                                remark-lint:awesome-list-item
```

**Comment by kdeldycke** [comment]
For the record, I started to specify the punctuation rules of quotes ar #101. Let's move the conversation there.
